src/app/feedback-form/feedback-form.component.html

```diff
@@ -18,9 +21,22 @@ <h3 class="contact-form__title">Get in Touch with Us!</h3>
           class="contact-form__input"
           type="email"
           id="email"
-          name="email"
-          required
+          formControlName="email"
         />
+        <div
+          class="error-message"
+          [hidden]="
+            !(form.get('email')?.touched && form.get('email')?.hasError('required') && !form.get('email')?.value)
+          "
+        >
+          Your Email is required
+        </div>
+        <div
+          class="error-message"
+          [hidden]="!(form.get('email')?.hasError('email') && !form.get('email')?.hasError('required'))"
```

Comment:
you should be explicit with your error definition. If there are more errors with the email it will be confusing for the user.